admin/partials/boldgrid-backup-admin-tools.php

```diff
@@ -28,6 +28,11 @@
 			'title'   => __( 'Cron Log', 'boldgrid-backup' ),
 			'content' => $this->core->cron_log->get_markup(),
 		),
+		array(
+			'id'      => 'section_logs',
+			'title'   => __( 'Logs', 'boldgrid-backup' ),
+			'content' => include BOLDGRID_BACKUP_PATH . '/admin/partials/tools/logs.php',
```

Comment:
The included file BOLDGRID_BACKUP_PATH . '/admin/partials/tools/logs.php' does not appear to have been committed to this branch. Unable to obtain logs. Receiving the following error:

PHP Warning:  include(): Failed opening '/home/bgdev/log.testdev.wolfindva.com/wp-content/plugins/boldgrid-backup/admin/partials/tools/logs.php' for inclusion (include_path='.:/opt/cpanel/ea-php73/root/usr/share/pear') in /home/bgdev/log.testdev.wolfindva.com/wp-content/plugins/boldgrid-backup/admin/partials/boldgrid-backup-admin-tools.php on line 34